### ISSUE TITLE
Unable to delete AppDeployToolkitBanner.png in some situations (Fix included)

### ISSUE THREAD

**Issue by That-Annoying-Guy** [PSAppDeployToolkit/PSAppDeployToolkit#717]
**Describe the bug**
see [Discourse post](https://discourse.psappdeploytoolkit.com/t/unload-appdeploytoolkitbanner-png/4014)
Created Bug Report as encouraged by [sintaxasn](https://discourse.psappdeploytoolkit.com/u/sintaxasn)

**To Reproduce**
- Copy of the package (AppDeployToolkit, Files and script) to %TEMP%
- Install package using the .ps1 script
- Remove the package from %TEMP%
- All at once in a single PowerShell session. The issue we have, is that the file AppDeployToolkit\AppDeployToolkitBanner.png cannot be deleted because it is still loaded by PowerShell,

**Toolkit Version:**
3.8.4 and 3.8.5 beta (June 2022)

**Powershell Version:**
5.1

**Additional context**
To fix, in  AppDeployToolkitMain.ps1 in the` # Calculate banner height` section.

add `$appDeployLogoBannerObject.Dispose()` as shown below:

```
# Calculate banner height
[Int32]$appDeployLogoBannerHeight = 0
try {
	[System.Drawing.Bitmap]$appDeployLogoBannerObject = New-Object System.Drawing.Bitmap $appDeployLogoBanner
	[Int32]$appDeployLogoBannerHeight = $appDeployLogoBannerObject.Height
	if ($appDeployLogoBannerHeight -gt $appDeployLogoBannerMaxHeight) {
		$appDeployLogoBannerHeight = $appDeployLogoBannerMaxHeight
	}
	$appDeployLogoBannerObject.Dispose()	#Must dispose() when install from local cache or else AppDeployToolkitBanner.png is locked open and cannot be removed when removing the pkg
}
catch { }
```

**Comment by seanlillis** [comment]
This will be fixed in the next release, thank you.

**Comment by seanlillis** [comment]
Fixed in v3.9.0.
